libs/dataset-explorer/src/lib/FeatureBalanceMeasuresChart.tsx

```diff
@@ -0,0 +1,232 @@
+            <Stack.Item>
+              <Text variant="large" className={styles.boldText}>
+                {measuresLocalization.Name}
+              </Text>
+            </Stack.Item>
+
+            <Stack.Item className={styles.callout}>
+              <LabelWithCallout
+                label=""
+                calloutTitle={measuresLocalization.Callout.Title}
+                type="button"
+              >
+                <Text block>{measuresLocalization.Callout.Description}</Text>
+                <Link
+                  // TODO: Replace link with https://responsibleaitoolbox.ai/ link once docs are published there
+                  href="https://microsoft.github.io/SynapseML/docs/features/responsible_ai/Data%20Balance%20Analysis/#feature-balance-measures"
+                  target="_blank"
+                >
+                  {localization.ModelAssessment.DataBalance.LearnMore}
+                </Link>
+              </LabelWithCallout>
+            </Stack.Item>
+          </Stack>
+        </Stack.Item>
+
+        {/* Renders the the two dropdowns, their respective headings, and the description */}
+        <Stack.Item>
+          <Stack horizontal tokens={{ childrenGap: "s1" }}>
+            {/* Renders the Feature dropdown and its heading */}
+            <Stack.Item>
+              <Stack tokens={{ childrenGap: "s1" }}>
+                <Stack.Item>
+                  <Text variant="mediumPlus" className={styles.boldText}>
+                    {measuresLocalization.FeaturePicker}
+                  </Text>
+                </Stack.Item>
+                <Stack.Item>
+                  <Dropdown
+                    styles={{
+                      callout: {
+                        selectors: {
+                          ".ms-Button-flexContainer": {
+                            width: "100%"
+                          }
+                        }
+                      },
+                      dropdown: {
+                        width: 150
+                      }
+                    }}
+                    id="featureDropdown"
+                    options={featureOptions}
+                    selectedKey={this.state.selectedFeatureIndex}
+                    onChange={this.setSelectedFeature}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+
+            {/* Renders the Measure dropdown and its heading */}
+            <Stack.Item>
+              <Stack tokens={{ childrenGap: "s1" }}>
+                <Stack.Item>
+                  <Text variant="mediumPlus" className={styles.boldText}>
+                    {measuresLocalization.MeasurePicker}
+                  </Text>
+                </Stack.Item>
+                <Stack.Item>
+                  <Dropdown
+                    styles={{
+                      callout: {
+                        selectors: {
+                          ".ms-Button-flexContainer": {
+                            width: "100%"
```

Comment:
I am not quite familiar with this. Can you explain what it does?

Reply:
It actually turns out that it's not necessary at all, only the `dropdown: width` is! 
Thank you for pointing this out, this is what I get for copying dropdown code from another part in the repo without asking what it does 😁